homeassistant/components/lektrico/strings.json

```
@@ -0,0 +1,101 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "description": "Set required parameters to connect to your device",
+        "data": {
+          "host": "[%key:common::config_flow::data::host%]",
+          "friendly_name": "[%key:common::config_flow::data::name%]"
+        }
+      },
+      "zeroconf_confirm": {
+        "description": "Do you want to add the Lektrico Charger with serial number `{serial_number}` to Home Assistant?",
+        "title": "Discovered Lektrico Charger device"
+      }
+    },
+    "error": {
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]"
+    },
+    "abort": {
+      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]",
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]"
+    }
+  },
+  "entity": {
+    "sensor": {
+      "state": {
+        "name": "State",
+        "state": {
+          "available": "Available",
+          "connected": "Connected",
+          "need_auth": "Waiting for Authentication",
```

```suggestion
          "need_auth": "Waiting for authentication",
```
Use sentence case in translation strings

---

I changed it, thanks.